Test/Platform/Tools/MFTestSystem/TestSystem.cs

```diff
@@ -61,12 +61,12 @@ static void Main(string[] args)
                 if (args[0] == "/?")
                 {
                     DisplayHelp();
-                    return;
+                    return -1;
                 }
                 else if (String.Equals(args[0], "-help", StringComparison.InvariantCultureIgnoreCase))
                 {
                     DisplayHelp();
-                    return;
+                    return -1;

```

Comment:
Displaying help should not produce an error code. (This can mess up CMD and bat scripts that set ERRORLEVEL based on the return code of the last run process)